src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design/build/netstandard1.6/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.targets

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
But the NuGet package ends up on a customer machine? If so then I would omit the header (and anywhere else we may have put this). We put the header only within our "source code" and not on anything that ends up as-is on a customer machine.